app/components/primer/beta/link.rb

```diff
@@ -47,7 +47,7 @@ def initialize(href: nil, scheme: DEFAULT_SCHEME, muted: false, underline: true,
           SCHEME_MAPPINGS[fetch_or_fallback(SCHEME_MAPPINGS.keys, scheme, DEFAULT_SCHEME)],
           "Link",
           "Link--muted" => muted,
-          "no-underline" => !underline
+          "Link--underline" => underline
```

Comment:
This seems like all links will always be underlined? Should we default `underline` to false then?

Reply:
I think underlining is the HTML default, which means we should default to `underline: true` also.